zhazam/Modules/Onboarding/View/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  OnboardingPageViewController.swift
+//  zhazam
+//
+//  Created by Nurbek Ismagulov on 4/19/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class OnboardingPageViewController: UIPageViewController {
+    
+    private var pages: [OnboardingPage]?
+    
+    private lazy var pageControl: UIPageControl = {
+        let pageControl = UIPageControl()
+        pageControl.numberOfPages = pages?.count ?? 0
+        pageControl.currentPage = 0
+        pageControl.currentPageIndicatorTintColor = R.color.textColor()
+        pageControl.pageIndicatorTintColor = R.color.defaultGray()
+        pageControl.translatesAutoresizingMaskIntoConstraints = false
+        return pageControl
+    }()
+    
+    init(with pages: [OnboardingPage]) {
+        super.init(transitionStyle: .scroll,
+                   navigationOrientation: .horizontal,
+                   options: nil)
+        self.pages = pages
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.dataSource = self
+        self.delegate = self
+        setInitialViewController()
+        setupPageControl()
+    }
+    
+    private func setInitialViewController() {
+        guard let viewController = getPage(for: 0) else { return }
+        setViewControllers([viewController],
+                           direction: .forward,
+                           animated: false,
+                           completion: nil)
+    }
+    
+    private func setupPageControl() {
+        view.addSubview(pageControl)
+        
+        let constraints = [
+            pageControl.bottomAnchor.constraint(equalTo: self.view.bottomAnchor, constant: -50),
+            pageControl.centerXAnchor.constraint(equalTo: self.view.centerXAnchor)
+        ]
+        NSLayoutConstraint.activate(constraints)
```

Comment:
Write NSLayoutConstraint.activate([...]) like sundell does, no need to instantiate a variable constants